module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,16 @@
+#include "sort.hpp"
+
+#include <algorithm>
+#include <forward_list>
+#include <string>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& flist) {
+    flist.sort([](const std::string& a, const std::string& b) { return a.size() == b.size() ? a < b : a.size() < b.size(); });
+
+    std::deque<std::string> dq;
+    for (auto el : flist) {
+        dq.push_back(el);
+    }
```

Comment:
You can initialaze deque like this:
std::deque<std::string> dq{flist.begin(), flist.end()};

Reply:
+1